docs/content.zh/docs/connectors/table/kafka.md

```diff
@@ -309,6 +309,31 @@ CREATE TABLE KafkaTable (
       <td>Long</td>
       <td>在使用 <code>'timestamp'</code> 启动模式时指定启动的时间戳（单位毫秒）。</td>
     </tr>
+    <tr>
+      <td><h5>scan.bounded.mode</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">unbounded</td>
+      <td>String</td>
```

Comment:
I guess this is not an enum because we still don't support `-` in names?

Reply:
It is enum, actually. It's just that these docs are not autogenerated and I guess it's been copied over from `scan.startup.mode` which has not been updated as well.